precommit/run-mirsg-hooks.py

```diff
@@ -2,14 +2,17 @@
 import pathlib
 import subprocess
 import sys
+import shlex
 
 HERE = pathlib.Path(__file__).resolve()
 
 
 def main() -> int:
     cfg = HERE.parent / "mirsg-hooks.yaml"
-    cmd = ["pre-commit", "run", "--config", f"{cfg}", "--files"] + sys.argv[1:]
-    result = subprocess.run(cmd, check=True)
+    result = subprocess.run(
+        shlex.split(f"pre-commit run --config {cfg} --files {sys.argv[1:]}"),
```

Comment:
```suggestion
        f"pre-commit run --config {cfg} --files".split()" + sys.argv[1:]
```
I think this will break things on Windows? May be better to keep previous style, or use a regular string for the command

Reply:
or maybe it's only the `quote` function that isn't supported? https://docs.python.org/3/library/shlex.html#shlex.quote

Reply:
reading again, it says
> The shlex module is only designed for Unix shells.

So probably best not to use. There's more discussion [here](https://stackoverflow.com/questions/33560364/python-windows-parsing-command-lines-with-shlex)

Reply:
Testing in Windows now (different repo) https://github.com/UCL-ARC/python-tooling/actions/runs/6651656121?pr=190

Reply:
That one didn't work ^ this one https://github.com/UCL-ARC/python-tooling/actions/runs/6651771962?pr=190

Reply:
looks like it's passing now? Is that because you removed `shlex`?

I imagine this one would fail too - it looks like the path on windows doesn't have any slashes to designate folders when you use shlex

Reply:
Have verified that this won't work on Windows have changed